src/codeyCommand.ts

```diff
@@ -0,0 +1,61 @@
+import { Command as SapphireCommand, container, SapphireClient } from '@sapphire/framework';
+import { Message, MessagePayload, WebhookEditMessageOptions } from 'discord.js';
+import { APIMessage } from 'discord-api-types/v9';
+import { isMessageInstance } from '@sapphire/discord.js-utilities';
+
+export type SapphireMessageRequest = APIMessage | Message<boolean>;
+export type SapphireMessageResponse = string | MessagePayload | WebhookEditMessageOptions;
+
+export type SapphireMessageExecuteType = (
+  client: SapphireClient<boolean>,
+  messageFromUser: Message | SapphireCommand.ChatInputInteraction,
+  initialMessageFromBot: SapphireMessageRequest
+) => SapphireMessageResponse;
+
+export type CodeyCommandOptions = {
+  name: string;
+  aliases: string[];
+  description: string;
+  detailedDescription: string;
+};
+
+export class CodeyCommand extends SapphireCommand {
```

Comment:
Hey Marcus! Everything looks awesome. 
Please just give a few more lines of comments/docs where things might be new/confusing so people can quickly change things for using slash-cmds